steamcmd_servers/valheim/valheim_plus/egg-valheim-plus-mod.json

```diff
@@ -0,0 +1,130 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v1",
+        "update_url": null
+    },
+    "exported_at": "2021-02-23T15:36:16+01:00",
+    "name": "Valheim Plus Mod",
+    "author": "info@goover.de",
+    "description": "A brutal exploration and survival game for 1-10 players, set in a procedurally-generated purgatory inspired by viking culture.",
+    "features": null,
+    "images": [
+        "quay.io\/parkervcp\/pterodactyl-images:debian_source"
+    ],
+    "startup": ".\/start_server_bepinex.sh",
```

Comment:
I am against using shell scripts to start things.

Reply:
> I am against using shell scripts to start things.

Thats the normal Startscript from the Modloader. is installed with the mod.

Reply:
LSGM use the same:

https://github.com/valheimPlus/ValheimPlus/issues/49#issuecomment-784275395

Reply:
fixed with latest commit